crates/valence_network/src/lib.rs

```diff
@@ -538,6 +553,18 @@ pub enum ServerListPing<'a> {
     Ignore,
 }
 
+/// The result of the Broadcast To Lan [callback].
+///
+/// [callback]: NetworkCallbacks::broadcast_to_lan
+#[derive(Clone, Default, Debug)]
+pub enum BroadcastToLan<'a> {
+    /// Disabled Broadcast To Lan.
+    #[default]
+    Disabled,
+    /// Send packet to broadcast to LAN every 1.5 seconds with specified MOTD.
+    Enabled(&'a str),
```

Comment:
I think using `&str` here isn't going to work if users wanted to create a message dynamically in the body of `broadcast_to_lan`, since the lifetime would end before returning. So it should be a `Cow<str>` instead.